.github/workflows/ci.yml

```diff
@@ -193,6 +193,15 @@ jobs:
           name: "${{ runner.os }}-bins"
           path: dist/bin
 
+      - name: Update PATH to include sAW
+        shell: bash
+        run: |
+          chmod +x dist/bin/*
+          echo $GITHUB_WORKSPACE/dist/bin >> $GITHUB_PATH
```

Comment:
Is this `$GITHUB_PATH` variable used? Seems like you use `$GITHUB_WORKSPACE/dist/bin` directly below.

Reply:
Yeah, there is some duplication of concerns here, because I was trying to get it to work. I'll fix some of that now...